docs/user_guide/SIMP_Administration/General_Administration.rst

```diff
@@ -0,0 +1,292 @@
+repositories than it is to manage individual package minutia for all packages
+across the environment.
+
+To use this effectively, packages that all systems will receive should be
+placed into the ``Updates`` repository provided with SIMP. Any packages that
+will only go to specific system sets should then be placed into adjunct
+repositories under ``/var/www/yum`` and the user will point specific systems at
+those repositories using the ``yumrepo`` Puppet Type. Any common packages can
+be either symlinked or hard linked between repositories for efficiency.
+
+
+Changing the Default Repositories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, SIMP stores :term:`YUM` information in the following directories:
+
+ - ``/var/www/yum``
+
+The base SIMP repository is in ``/var/www/yum/SIMP`` and it is highly unlikely
+that you would want to modify anything in this directory.
+
+By default, access to the YUM repository is restricted to the networks
+contained in the ``simp_options::trusted_nets`` parameter. For this section, we
+will assume that this is sufficient.
+
+
+The Operating System Repos
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default location for the :term:`Operating System` (OS) repositories, on the
+Puppet server, is ``/var/www/yum/<OSTYPE>/<MAJORRELEASE>/x86_64``.
+
+An ``Updates`` repository has been configured in this space. All OS updates
+should be placed within this directory.
+
+You should run the following in the ``Updates`` directory after **ANY** package
+addition or removal within that directory.
+
+.. code-block:: bash
+
+  $ createrepo .
+  $ chown -R root.apache ./*
+  $ find . -type f -exec chmod 640 {} \;
+  $ find . -type d -exec chmod 750 {} \;
+
+
+Adding a Custom Repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For this section, we will assume that you have a repository named ``foo`` that
+you would like to expose to your systems. To do this, perform the following:
+
+  .. code-block:: bash
+
+     $ cd /var/www/yum
+     $ mkdir foo
+     $ cd foo
+     $ -- copy all RPMs into the folder
+     $ createrepo .
+     $ chown -R root.apache ./*
+     $ find . -type f -exec chmod 640 {} \;
+     $ find . -type d -exec chmod 750 {} \;
+
+.. NOTE::
+
+   For more information on managing YUM repos, please see the
+   `Red Hat local repository Documentation`_.
+
+
+Configuring the Clients
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Now that you've added this repository, you're going to want to add it to your
+clients.
+
+The best way to do this is to make it part of your site profile. You **can**
+make it part of your module, but you will need to wrap it in a Defined Type so
+that the server parameter can be modified.
+
+To add it to your clients, use the puppet ``yumrepo`` Type. You can find more
+information in the `Puppet Type Reference`_.
+
+The following is a basic ``yumrepo`` example:
+
+.. code-block:: ruby
+
+  yumrepo { example:
+    baseurl         => "http://your.server.fqdn/yum/example",
```

Comment:
do you want to use 'foo' instead of 'example' to tie this in to the previous section?